sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -122,6 +123,33 @@ object JdbcUtils extends Logging {
     }
   }
 
+  /**
+   * Runs a custom query against a table from the JDBC database.
+   */
+  def runQuery(conn: Connection, actions: String, options: JDBCOptions): Unit = {
+    val autoCommit = conn.getAutoCommit
+    conn.setAutoCommit(false)
+
+    val commands = QueryStatementUtils.splitSemiColon(actions).asScala
+    val statement = conn.createStatement()
+    statement.setQueryTimeout(options.queryTimeout)
+
+    commands.foreach { command =>
```

Comment:
How about adding `logInfo` here? https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala#L283

Reply:
Added `logInfo` in similar way.